extensions/libray-occupancy/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "libray-occupancy",
+  "title": "Main Library in University of St Andrews",
+  "description": "How busy is the main library now?",
+  "icon": "command-icon.png",
+  "author": "Harry",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Library Occupancy",
+      "description": "Template with a plain detail view",
```

Comment:
This probably was missed when copying over the template

Reply:
That would indeed make sense to change